### DISCUSSION TITLE
"Error: Cannot read property 'commit' of undefined"

### DISCUSSION THREAD

**Discussion by leahcimto** [upptime/upptime#284] [Q&A] [answered]
Hello!
I made a [repository](https://github.com/leahcimto/forums-uptime) with a site I would like to track. I configured everything in getting started, and it says it should update README.md, however, I got the error "Cannot read property 'commit' of undefined"
<img width="991" alt="Screen Shot 2021-03-11 at 5 26 51 PM" src="https://user-images.githubusercontent.com/68165163/110863640-f9a15980-828e-11eb-820a-44c52dcf4c03.png">

Any help would be appreciated!

**Comment by Hans5958** [comment, marked as answer]
I also got this issue a while ago, but I fixed it by giving my bot a write permission by adding him on the repo, since I use the token of the said account for it.

**Reply by leahcimto** [reply]
Thank you so much! This fixed it.
